tests/database/test.sh

```diff
@@ -0,0 +1,224 @@
+    last_substring=$(echo "${last_name:1}" | tr '[:upper:]' '[:lower:]')
+
+    folder_name="$first_name$first_letter$last_substring"
+}
+
+set_folder
+
+submission_directory="$PWD/../../submissions"
+path_to_folder="$submission_directory/$folder_name"
+path_to_database_folder="$path_to_folder/database"
+path_to_log="$submission_directory/../tests/logfile.json"
+
+check_database_folder_existence() {
+    folder_count=$(ls $path_to_folder | grep "database" -c)
+    if [ $folder_count -eq 0 ]
+    then
+        database_folder_exists=0
+        echo ""
+        echo -e "${BOLD}Checking the database folder ...${NONE}"
+        echo -e "${RED}Your folder should be named ${BOLD}\"database\"${NONE} ${RED}and located in the folder with your First and Last name. (${BOLD}\"$folder_name/database\"${NONE}${RED})${NONE}"
+    else
+        database_folder_exists=1
+    fi
+}
+
+check_file_existence() {
+    if [ -z "$(ls -A $path_to_database_folder)" ]
+    then
+        file_exists=0
+        echo ""
+        echo -e "${BOLD}Checking the text file ...${NONE}"
+        echo -e "${RED}1. You should create a text file."
+        echo -e "2. Ensure your sql file is located in the ${BOLD}\"database\"${NONE} ${RED}folder you have created."
+        echo -e "3. Ensure your sql file contains text."${NONE}
+    else
+        for currentFile in $path_to_database_folder/*
+        do
+            if [ $(file -b --mime-type "$currentFile") == "text/plain" ]
+            then
+                file_exists=1
+                break
+            else
+                file_exists=0
+                echo ""
+                echo -e "${BOLD}Checking the sql file ...${NONE}"
+                echo -e "2. Ensure your sql file is located in the ${BOLD}\"database\"${NONE} ${RED}folder you have created."
+            fi
+        done
+    fi
+}
+
+check_file_content1() {    
+    for currentFile in $path_to_database_folder/*
+    do
+        if grep  "CREATE TABLE admins"  $currentFile  && grep "CREATE TABLE vendor_food" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo ""
+            echo -e "1. Ensure your sql file contains the required SQL queries."${NONE}
+        fi
+   done
+}
+
+check_file_content2() {    
+    for currentFile in $path_to_database_folder/*
+    do
+        if grep  "CREATE TABLE orders"  $currentFile && grep "CREATE TABLE food" $currentFile
+        then
+            content_exists=1
+            break     
+        else
+            content_exists=0
+            echo -e "${BOLD}Checking the sql file ...${NONE}"
+            echo -e "1. Ensure your sql file contains the required SQL queries."${NONE}
+        fi
+   done
+}
+
+check_file_content3() {    
+    for currentFile in $path_to_database_folder/*
+    do
+        if grep  "id INT NOT NULL AUTO_INCREMENT"  $currentFile && grep "name VARCHAR NOT NULL" $currentFile
```

Comment:
I think it should just be something like this `name VARCHAR` only since grep will try and match the number specified by the scholar and the query we have here does not have number.